amazon-s3/5.7/modules/ROOT/pages/amazon-s3-connector-studio.adoc

```diff
@@ -76,19 +106,75 @@ Validating a connection with *Test Connection* requires that you have permission
 +
 Access to operations on Amazon S3 is further controlled through policies. It is not always possible to validate your credentials before the exact operation for which you have access completes. This can vary based on the bucket name and other parameters. For example, the test connection can fail if your credentials have a restricted policy.
 
-. Click *OK* to save the global connector configurations.
+[[basic-authentication]]
+=== Basic Authentication
+
+Enter the following information on the *General* tab of the *Global Element Properties* screen to configure Basic authentication:
+
+[%header,cols="30s,70a"]
```

Comment:
I like that you ordered this like it is on the screen. Just to check, I looked at v5.7 in Exchange, and the Basic configuration isn't showing the Try Default AWS Credentials Provider Chain field. It is now on the Advanced tab. This is probably a newer thing. To deal with this, you can add a new bullet saying something like this:

On the *Advanced* tab...